src/app/main/component/events/components/events-list/events-list.component.html

```diff
@@ -13,7 +13,7 @@
           <img class="my-events-img" src="assets/events-icons/my-event.png" alt="my-event" />
         </div>
         <div class="create">
-          <div tabindex="0" class="secondary-global-button m-btn" (click)="isUserLoggedRedirect()">
+          <div tabindex="0" class="secondary-global-button m-btn" (keydown.Tab)="isUserLoggedRedirect()" (click)="isUserLoggedRedirect()">
```

Comment:
As an alternative, you can replace `<div>` with `<button>` - so it will be clickable on Tab without adding (keydown.Tab)

Reply:
Thank you! Add changes!